test/mongoose_push_application_test.exs

```diff
@@ -0,0 +1,63 @@
+defmodule MongoosePushApplicationTest do
+  use ExUnit.Case
+  import MongoosePush.Application
+  doctest MongoosePush.Application
+
+  test "worker name" do
+    assert :apns_name1_1 == worker_name(:apns, :name1, 1)
+    assert :fcm_name2_12 == worker_name(:fcm, :name2, 12)
+  end
+
+  test "pool groups" do
+    assert [:dev1, :dev2] == pools_by_mode(:apns, :dev)
+    assert [:prod1, :prod2] == pools_by_mode(:apns, :prod)
```

Comment:
Ahh I know, there's the `config/test.exs`. Hmm if someone changes this file the test won't pass and assertion error will be printed. Seems to me that then the error becomes not obvious. One way of tackling this would be to have simple assertion like `Application.get_env(:mongoose_push, :sth)[:pools] == 4` (that's pseudo code ofc)